static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -3505,14 +3505,12 @@ const initializeCStudioForms = () => {
               'createdDate_dt',
               'lastModifiedDate',
               'lastModifiedDate_dt',
-              'components',
```

Comment:
Per conversation with Russ, this `components` field is not part of the initial valid fields, since it's a legacy field we used for components inside RTEs. This was causing the stale `components` field to stay in the xml.